### PR TITLE
Changed corefx to libraries in System.Private.CoreLib

### PR DESCRIPTION
**Pull request by eanova**
This PR is related to #347 

These are the simple changes I could make to this folder,  updated links and comments.

### CODE REVIEW

File: src/libraries/System.Private.CoreLib/src/System/Runtime/Serialization/SerializationInfo.cs [hunk 1]
```diff
@@ -402,13 +402,13 @@ internal void AddValueInternal(string name, object? value, Type type)
         /// ObjectManager when it's performing fixups.
         ///
         /// All error checking is done with asserts. Although public in coreclr,
-        /// it's not exposed in a contract and is only meant to be used by corefx.
+        /// it's not exposed in a contract and is only meant to be used by libraries.
```
Comment (stephentoub) [inline src/libraries/System.Private.CoreLib/src/System/Runtime/Serialization/SerializationInfo.cs]:
```suggestion
        /// it's not exposed in a contract and is only meant to be used by other runtime libraries.
```

File: src/libraries/System.Private.CoreLib/src/System/Runtime/Serialization/SerializationInfo.cs [hunk 2]
```diff
@@ -402,13 +402,13 @@ internal void AddValueInternal(string name, object? value, Type type)
...
         ///
         /// This isn't a public API, but it gets invoked dynamically by
         /// BinaryFormatter
         ///
         /// This should not be used by clients: exposing out this functionality would allow children
-        /// to overwrite their parent's values. It is public in order to give corefx access to it for
+        /// to overwrite their parent's values. It is public in order to give libraries access to it for
```
Comment (stephentoub) [inline src/libraries/System.Private.CoreLib/src/System/Runtime/Serialization/SerializationInfo.cs]:
```suggestion
        /// to overwrite their parent's values. It is public in order to give other runtime libraries access to it for
```

File: src/libraries/System.Private.CoreLib/src/Interop/Unix/Interop.Errors.cs [hunk 3]
```diff
@@ -105,7 +105,7 @@ internal enum Error
         EHOSTNOTFOUND    = 0x20001,           // Name lookup failed
 
         // POSIX permits these to have the same value and we make them always equal so
-        // that CoreFX cannot introduce a dependency on distinguishing between them that
+        // that libraries cannot introduce a dependency on distinguishing between them that
```
Comment (stephentoub) [inline src/libraries/System.Private.CoreLib/src/Interop/Unix/Interop.Errors.cs]:
```suggestion
        // that we do not introduce a dependency on distinguishing between them that
```

File: src/libraries/System.Private.CoreLib/src/Internal/Win32/RegistryKey.cs [hunk 4]
```diff
@@ -16,7 +16,7 @@
 // A minimal version of RegistryKey that supports just what CoreLib needs.
 //
 // Internal.Win32 namespace avoids confusion with the public standalone Microsoft.Win32.Registry implementation
-// that lives in corefx.
+// that lives in libraries.
```
Comment (stephentoub) [inline src/libraries/System.Private.CoreLib/src/Internal/Win32/RegistryKey.cs]:
```suggestion
// that lives in https://github.com/dotnet/runtime/tree/master/src/libraries/Microsoft.Win32.Registry
```

**Review by stephentoub — COMMENTED** [review]
A few suggestions, otherwise LGTM. Thanks.

**Comment by eanova** [comment]
Thanks for the suggestions.  I'll make sure to use "runtime libraries".  Where I got confused is thinking that "libraries" was a new branding for CoreFx.

**Review by stephentoub — APPROVED** [review]

**Pull request merged**